console.py

```diff
@@ -118,10 +118,22 @@ def do_create(self, args):
         if not args:
             print("** class name missing **")
             return
-        elif args not in HBNBCommand.classes:
+        args = args.split()
+        if args[0] not in HBNBCommand.classes:
             print("** class doesn't exist **")
             return
-        new_instance = HBNBCommand.classes[args]()
+        new_instance = HBNBCommand.classes[args[0]]()
+        storage.save()
+        obj = storage.all()
+        if args[1]:
+            idnum = new_instance.id
+            for i in range(1, len(args)):
+                param = args[i].split("=")
+                key = param[0]
+                value = param[1]
```

Comment:
Unless I'm missing something, these two variables are never used. I'm also not sure they are necessary if we can just call param[0] and param[1].